lib/iris/tests/unit/analysis/cartography/test_rotate_grid_vectors.py

```diff
@@ -71,7 +71,7 @@ def _check_angles_calculation(
         ang_diffs = out_angs - expect_angs
         # Fix for null vectors, and +/-360 differences.
         ang_diffs[np.abs(out_mags) < 0.001] = 0.0
-        ang_diffs = ang_diffs % 360.0
+        ang_diffs[np.isclose(np.abs(ang_diffs), 360.0)] = 0.0
```

Comment:
Same result, but more robust given the floating point numbers we're dealing with.